velox/connectors/hive/HiveConnectorUtil.cpp

```diff
@@ -251,6 +251,19 @@ bool isSpecialColumn(
   return specialName.has_value() && name == *specialName;
 }
 
+bool supportsMergeWithOr(const common::FilterPtr& filter) {
+  switch (filter->kind()) {
+    case common::FilterKind::kBigintValuesUsingHashTable:
+    case common::FilterKind::kBigintValuesUsingBitmask:
+    case common::FilterKind::kNegatedBigintRange:
```

Comment:
Is there a reason why negated ranges cannot be combined into BigintMultiRange ? Is this just a bug / missing functionality in the filters?